tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "target": "ES2021",
+    "outDir": "dist",
+    "module": "commonjs",
+    "skipLibCheck": true,
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "lib": ["ES2021", "dom"],
+    "rootDir": "src"
+  },
+  "include": ["./src/**/*"]
```

:slightly_smiling_face: 

```suggestion
  "include": ["src"]
```

BTW, why not copy tsconfig.json from relayer where strict mode is already configured and other things?

---

I have updated from relayer and added  `"declaration": true,` also included your change.